src/fd.zig

```diff
@@ -170,14 +170,29 @@ pub const FDImpl = packed struct {
         return switch (env.os) {
             else => numberToHandle(this.value.as_system),
             .windows => switch (this.kind) {
-                .system => std.debug.panic(
-                    \\Cast {} -> FDImpl.UV makes closing impossible!
-                    \\
-                    \\The supplier of this FileDescriptor should call 'bun.toLibUVOwnedFD'
-                    \\or 'FDImpl.makeLibUVOwned', probably where open() was called.
-                ,
-                    .{this},
-                ),
+                .system => {
+                    const w = std.os.windows;
+
+                    const S = struct {
+                        fn is_stdio_handle(id: w.DWORD, handle: w.HANDLE) bool {
```

Comment:
instead of declaring a struct with a function here, why not just move the function above `pub fn uv`?